### PR TITLE
Members

### PR DESCRIPTION
**Pull request by bialy358**
- [x] add member model  
- [x] CRUD  for members 
- [x] auth for members/ owner actions  
- [ ] SPECS

### CODE REVIEW

File: app/controllers/control_panel/boards_controller.rb [hunk 1]
```diff
@@ -1,11 +1,12 @@
 class ControlPanel::BoardsController < ControlPanel::ControlPanelController
+  before_action :require_owner, only: [:edit, :update, :destroy]
 
   def index
-    @boards = current_user.boards
+    @boards = current_user.boards + Board.of_members(current_user)
```
Comment (PatrykTyszka) [inline app/controllers/control_panel/boards_controller.rb:5]: WTF ? :D

**Comment by bialy358** [comment]
kurwa mac jak ja to zrobilem :dancer:

File: app/forms/member_form.rb [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    else
+      false
+    end
+  end
+
+  private
+  def owner?
+    user = User.find_by(email: email)
+    board = Board.find(board_id)
+    unless user
```
Comment (PiotrProtas) [inline app/forms/member_form.rb:27]: `if/else`. Think about proper order

**Pull request merged**